fitwsarah_frontend/src/views/AboutMe/AboutMe.js

```diff
@@ -0,0 +1,103 @@
+import { useState, useEffect } from "react";
+import { useAuth0 } from '@auth0/auth0-react';
+import "../../css/style.css";
+import NavNotLoggedIn from "../../components/navigation/NotLoggedIn/navNotLoggedIn";
+import FooterNotLoggedIn from "../../components/footer/footer";
+import NavLoggedIn from "../../components/navigation/loggedIn/navLoggedIn";
+import { ROLES } from "../../components/authentication/roles";
+import "../../components/authentication/switch.css"
+import RoleBasedSwitch from "../../components/authentication/RoleBasedSwitch";
+import 'react-toastify/dist/ReactToastify.css';
+import ReactStars from 'react-stars'
+import AddFeedbackButton from '../../components/feedback/newFeedbackBtn';
+import "./AboutMe.css";
+import { useTranslation } from "react-i18next";
+import workoutImage from './workout.png';
+import trainerImage from './trainer.png';
+function AboutMe() {
+    const {
+        isAuthenticated,
+        user
+    } = useAuth0();
+
+    const [editMode, setEditMode] = useState(false);
+    const {t} = useTranslation('aboutMe');
+    const {sub} = isAuthenticated ? user : {};
+
+
+    function extractAfterPipe(originalString) {
+        const parts = originalString.split('|');
+        if (parts.length === 2) {
+            return parts[1];
+        } else {
+            return originalString;
+        }
+    }
+
+    // Define a CustomImg component that applies a class name to the rendered img tag
+    function CustomImg({ src, alt, width, height, className }) {
+        // Include the `className` prop to allow custom class names
+        return <img src={src} alt={alt} width={width} height={height} className={className} />;
+    }
+
+
+
+
+
+
+    return (
+        <div>
+            {!isAuthenticated && <NavNotLoggedIn />}
+            {isAuthenticated && <NavLoggedIn />}
+            <div id="contactBackground">
+                <div className="container1">
+                    <div className="box">
+                        <p className="pargah1">{t('hello')}<br />
+                            <span className="large-bold">{t('name')},</span><br />
+                            {t('info')}</p>
+                    </div>
+                    <div className="box">
+                        <div className="image-container">
+                            <CustomImg src={workoutImage} alt="Workout" width="400" height="400" className="custom-img" />
+                        </div>
+                    </div>
+                </div>
```

Comment:
Why did you create custom img tag the normal wouldn't work?

Reply:
Good question, because the normal tag refenrce to the img tag in the navlogged in file and affected the logo size.